setup.cfg

```diff
@@ -7,7 +7,7 @@ tag = false
 
 [bumpversion:file:sumy/__init__.py]
 
-[pytest]
+[tool:pytest]
 addopts = --quiet --tb=short --color=yes --cov=sumy --cov-report=term-missing --no-cov-on-fail
```

Comment:
@miso-belica you have pytest-cov options listed here, so anyone invoking `pytest` command (without any options) will have to have pytest-cov installed. So it is not optional. Thus, it's either removing pytest-cov options from here or make it required as a dependency.

Reply:
@kmkurn Thanks for this. Then make pytest-cov required please.